src/java/org/apache/cassandra/db/virtual/SettingsTable.java

```diff
@@ -59,63 +53,77 @@ final class SettingsTable extends AbstractVirtualTable
                            .addPartitionKeyColumn(NAME, UTF8Type.instance)
                            .addRegularColumn(VALUE, UTF8Type.instance)
                            .build());
-        this.config = config;
+        Set<String> unprocessed = new HashSet<>(BACKWARDS_COMPATABLE_NAMES.values());
+        DatabaseDescriptor.visit((key, type, ro) -> {
+            if (BACKWARDS_COMPATABLE_NAMES.containsKey(key))
+            {
+                throw new AssertionError(String.format("Name '%s' is present in Config, this adds a conflict as this " +
+                                                       "name had a different meaning in the '%s",
+                                                       key, SettingsTable.class.getSimpleName()));
+            }
+            unprocessed.remove(key);
+        });
+        if (!unprocessed.isEmpty())
+            throw new AssertionError("The following keys must be present in the Config: " + unprocessed);
+    }
+
+    @Override
+    protected void applyColumnDeletion(AbstractMutableVirtualTable.ColumnValues partitionKey, AbstractMutableVirtualTable.ColumnValues clusteringColumns, String columnName)
```

Comment:
pointed this out in `Config`; not all properties support `DELETE`, all support `INSERT` but only a small subset support `DELETE`... I would recommend dropping `DELETE` support for this patch to keep things simple... as you need to understand what are the semantics for `DELETE`... is `DELETE` "use default" or "disable" or something else?

Reply:
Support of the `DELETE` operation was removed.